src/main/java/org/skriptlang/reflect/syntax/condition/elements/StructCustomCondition.java

```diff
@@ -0,0 +1,186 @@
+package org.skriptlang.reflect.syntax.condition.elements;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.ClassInfo;
+import ch.njol.skript.config.Node;
+import ch.njol.skript.config.SectionNode;
+import ch.njol.skript.lang.Condition;
+import ch.njol.skript.lang.Literal;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.lang.SyntaxElementInfo;
+import ch.njol.skript.lang.Trigger;
+import ch.njol.skript.lang.TriggerItem;
+import ch.njol.skript.lang.util.SimpleEvent;
+import ch.njol.skript.log.SkriptLogger;
+import ch.njol.skript.util.Utils;
+import org.skriptlang.reflect.syntax.CustomSyntaxStructure;
+import com.btk5h.skriptmirror.skript.custom.SyntaxParseEvent;
+import com.btk5h.skriptmirror.util.SkriptUtil;
+import org.skriptlang.reflect.syntax.condition.ConditionCheckEvent;
+import org.skriptlang.reflect.syntax.condition.ConditionSyntaxInfo;
+import org.skriptlang.skript.lang.entry.EntryContainer;
+import org.skriptlang.skript.lang.script.Script;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.function.Supplier;
+import java.util.stream.Collectors;
+
+public class StructCustomCondition extends CustomSyntaxStructure<ConditionSyntaxInfo> {
+
+  public static boolean customConditionsUsed = false;
```

Comment:
Where is this used? It probably needs to be updated to support unloading, right?

Reply:
I believe just for bstats